src/kruskal_wallis.jl

```diff
@@ -32,6 +32,41 @@ immutable KruskalWallisTest <: HypothesisTest
     tie_adjustment::Float64  # adjustment for ties
 end
 
+"""
+    KruskalWallisTest(groups::AbstractVector{<:Real}...)
+
+Perform Kruskal-Wallis rank sum test of the null hypothesis that the `groups`
+``\\mathcal{G}`` come from the same distribution against the alternative hypothesis that
+that at least one group stochastically dominates one other group.
+
+The Kruskal-Wallis test is an extension of the Mann-Whitney U test to more than two groups.
+
+The p-value is computed using a ``χ^2`` approximation to the distribution of the test
+statistic ``H_c=\\frac{H}{C}``:
+```math
+    \\begin{align}
```

Comment:
Good point, `align` numbers (although the Jupyter notebook doesn't print them). I'll change it to `align*` in the next iteration.